### PR TITLE
app-text/sigil: fix sed failure when using ninja instead of make

### PR DESCRIPTION
**Pull request by tomboy-64**
Toralf discovered a build failure when cmake chooses ninja over make.
This fixes it.
Respective bug report: https://bugs.gentoo.org/show_bug.cgi?id=562142

@monsieurp would you please be so kind to merge it? Thanks.

### CODE REVIEW

File: app-text/sigil/sigil-0.8.900_pre-r1.ebuild [hunk 1]
```diff
@@ -63,8 +63,9 @@ src_configure() {
 	cmake-utils_src_configure
 
 	cd "${WORKDIR}/${P}_build" || die "Where is the build dir?"
-	for i in CMakeCache.txt src/CMakeFiles/sigil.dir/link.txt $(find . -name '*.make')
+	for i in $(grep -Rl "O3 -DNDEBUG" .)
```
Comment (mgorny) [inline app-text/sigil/sigil-0.8.900_pre-r1.ebuild]: Wait, whaaaaaat!? I'd really prefer if we patched CMake source files, not random output they generate :-P.

Comment (mgorny) [inline app-text/sigil/sigil-0.8.900_pre-r1.ebuild]: Oh, I guess that's because of the CMAKE_BUILD_TYPE override. I wonder if that's good for anything.

**Comment by mgorny** [comment]
@gentoo/kde, could you try to help figure out how to handle this better? :-P

**Comment by mgorny** [comment]
…and @gentoo/proxy-maint as the assignment!

**Comment by tomboy-64** [comment]
@mgorny I made the requested changes and it didn't break anything obvious. Please merge.

**Comment by mgorny** [comment]
Ok, this looks better. However, this is a potentially breaking change, so I'd prefer if you did it with revbump — just on the newest version, ignore older ones. Afterwards, please improve the commit message to explain what you did and why. And please follow the git rules for commit messages (i.e. one-line summary must fit on one line).

**Comment by tomboy-64** [comment]
Okay, but since 0.8.900 is really an early preview, i still update 0.8.7 as well. This is the current stable version that people should be using. 0.8.900 also has a warning regarding this.

**Comment by prometheanfire** [comment]
http://chris.beams.io/posts/git-commit/ specifically the 7 rules (and rule 5 in particular) would be good, but other then that lgtm

**Comment by idella** [comment]
I usually add;   revbump to versions a & b then mb blank line and improved version of your msg, then I'll commit

**Comment by tomboy-64** [comment]
Thanks. I really needed that to rtfm.
Brb.

**Comment by tomboy-64** [comment]
@prometheanfire @idella is this acceptable?
When should I start pruning out old versions?

**Comment by prometheanfire** [comment]
It's a bit better, though the message could still use some work.  Sorry if we are being hard about this, but a good message is like good hygiene :P

as far as how much to keep around, I prefer to keep 2 stable and 1 unstable around, though sometimes that's just two stable branches (2.1.19 and 2.2.6 for example) if upstream works like that.  It's more dependant on what upstream is actively supporting than anything.

https://wiki.gentoo.org/wiki/Gentoo_git_workflow#Commit_message_format

**Comment by idella** [comment]
holding off until re-submission to better commit msg

**Comment by tomboy-64** [comment]
@prometheanfire could you review again please?
Took a bit longer cause of additional work I did on it.

If okay, please merge.

**Comment by prometheanfire** [comment]
message is very nice, I'll review/merge soonish then

File: app-text/sigil/sigil-0.8.901_pre.ebuild [hunk 2]
```diff
@@ -23,10 +23,17 @@ KEYWORDS="~amd64 ~x86"
...
 
 RDEPEND="
-	dev-lang/python:3.4
-	>=dev-libs/boost-1.49[threads]
-	>=dev-libs/libpcre-8.31[pcre16]
-	>=dev-libs/xerces-c-3.1.1[icu]
+	dev-libs/boost[threads]
+	dev-libs/libpcre[pcre16]
+	dev-libs/xerces-c[icu]
+	dev-python/chardet
```
Comment (mgorny) [inline app-text/sigil/sigil-0.8.901_pre.ebuild]: All dependencies on Python packages need `[${PYTHON_USEDEP}]` to request matching Python versions.

**Comment by prometheanfire** [comment]
have you signed the commit (git commit -S)?

remote: *** No signature on 64c08af0b7fe52e51db354fb5a7a5c9ce7e1ab83, refusing

**Comment by tomboy-64** [comment]
sorry, those were outdated. this one is proper.

**Comment by prometheanfire** [comment]
it's already been merged

**Comment by tomboy-64** [comment]
thanks. gonna tidy up the boost-python thing next revbump then.
thanks for your help.

**Pull request closed without merging**